piikii-application/src/main/kotlin/com/piikii/application/port/input/dto/response/CourseResponse.kt

```diff
@@ -0,0 +1,64 @@
+package com.piikii.application.port.input.dto.response
+
+import com.piikii.application.domain.course.CoursePlace
+import com.piikii.application.domain.room.Room
+import com.piikii.application.domain.schedule.Schedule
+import com.piikii.application.domain.schedule.ScheduleType
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "코스 정보 응답")
+data class CourseResponse(
+    @field:Schema(description = "모임 이름", example = "강남으로 모이자")
+    val roomName: String,
+    @field:Schema(description = "코스 장소 정보 목록", example = "사당으로 모이자")
+    val places: List<CoursePlaceResponse>,
+) {
+    companion object {
+        fun from(
+            room: Room,
+            placeBySchedule: Map<Schedule, CoursePlace>,
```

Comment:
`courseBySchedule`일 것 같습니다. 맞겠죠...?
```suggestion
            courseBySchedule: Map<Schedule, CoursePlace>,
```

이런 네이밍이 조금씩 보이는데 한번 쓱 다 훑어봐주시면 좋을 것 같아요 👍🏻

Reply:
앗 이거 Schedule과 대응되는 Place 찾아 매핑한 데이터입니다.
CoursePlace는 코스 후보로 선택된 장소(place) 데이터에요! 장소 내용을 중심으로 담고 있어 placeBySchedule로 명명했습니닷.

Reply:
아하 이해했습니다. 결국엔 Course에 해당하는 place여서 네이밍이 저거군요~! 👍🏻 
감사합니당ㅎㅎ